tests/lib/rules/no-current-route-name.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+const rule = require('../../../lib/rules/no-current-route-name');
+const RuleTester = require('eslint').RuleTester;
+
+const { ERROR_MESSAGE } = rule;
+
+const ruleTester = new RuleTester({
+  parserOptions: { ecmaVersion: 6, sourceType: 'module' },
+});
+
+ruleTester.run('no-current-route-name', rule, {
+  valid: [
+    "assert.equal(currentURL(), '/foo');",
+
+    // who knows...
+    `
+      import { currentURL as currentRouteName } from '@ember/test-helpers';
+
+      assert.equal(currentRouteName(), '/foo');
+    `,
+  ],
```

Comment:
`currentRouteName()` by itself (without import) would be a good valid test case.